.env.sample

```diff
@@ -3,3 +3,4 @@ GOOGLE_API_KEY=XXXXXXXX-XX
 NODE_ENV=development
 AUTH_USER=
 AUTH_PASSWORD=
+API_URL=
```

Comment:
Please tell me what should I put here. Furthermore, I think it wouldn't hurt to put the standard default value on this file.